rails-commenting.rb

```diff
@@ -3,28 +3,32 @@
 # Add comments to the Rails Blog Post Challenge
 # Explain the purpose and functionality of the code directly below the 10 comment tags
 
-
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+// BlogPostsController inherets from the ApplicationController
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    // we set a param named posts thats going to equall all the BlogPost
     @posts = BlogPost.all
   end
 
   # ---3)
+  //this creates the show method to find the selected post to be displayed
   def show
     @post = BlogPost.find(params[:id])
   end
 
   # ---4)
+  // the new method cretes a new BlogPost
```

Comment:
Remember that 'new' will display a form for the user, not actually create a new blog post